sdks/java/io/amqp/pom.xml

```diff
@@ -39,6 +39,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
-1: this dependency is not for testing: it's actually used for the login. The implementation is for testing.